lib/linter-reek.js

```diff
@@ -90,13 +90,15 @@ export default {
         while (match !== null) {
           const line = Number.parseInt(match[1], 10) - 1;
           const urlBase = 'https://github.com/troessner/reek/blob/master/docs/';
-          const ruleLink = `[<a href="${urlBase}${match[3]}.md">${match[3]}</a>]`;
+          const ruleLink = `${urlBase}${match[3]}.md`;
           messages.push({
-            filePath,
-            type: 'Warning',
+            url: ruleLink,
             severity: 'warning',
-            html: `${match[2]} ${ruleLink}`,
-            range: helpers.generateRange(TextEditor, line),
+            excerpt: `${match[2]}`,
```

Comment:
We're missing the rule name entirely in the messages now, I'd recommend changing this to `${match[2]} (${match[3]})` or something similar.

Reply:
That could be recurrent, the excerpt includes the human readable rule name:

`InstanceVariableAssumption: Dummy assumes too much for instance variable @ivar`

vs

`(Instance-Variable-Assumption) InstanceVariableAssumption: Dummy assumes too much for instance variable @ivar`

Reply:
Oh interesting, so it was duplicated before then as well? If that's the case let's just leave it as is.